code/game/objects/items/stacks/throwing.dm

```diff
@@ -80,3 +81,128 @@
 	var/ROB_throwing_damage = max(C.stats.getStat(STAT_ROB), 1)
 	throwforce = 35 / (1 + 100 / ROB_throwing_damage) + initial(throwforce) //soft cap; This would result in knives doing 10 damage at 0 rob, 20 at 50 ROB, 25 at 100 etc.
 	..()
+
+/////HUNTERS/////
+
+/obj/item/stack/thrown/throwing_knife/bone_needle
+	name = "throwing bone needle"
+	desc = "Sharp and light chiseled bone needles designed for throwing. They have a high penetrating power, but only in cases where there is no armor between the flesh and their tip.\
+	 They easily get stuck in wounds!"
+	icon = 'icons/obj/stack/items.dmi'
+	icon_state = "needle"
+	singular_name = "bone needle"
+	plural_name = "bone needles"
+	price_tag = 0
+	sharp = TRUE
+	edge = TRUE
+	embed_mult = 90
+	tool_qualities = list()
+	attack_verb = list("flies in", "stabbed", "pierces", "torn", "penetrates")
```

Comment:
Aren't these for melee attacks? The grammar may not be correct, it may need to be adjusted to account for someone using the needles as a weapon(e.g stabs, pierces, tears)